stream_alert/alert_processor/outputs/pagerduty.py

```diff
@@ -259,13 +264,12 @@ def _check_exists(self, filter_str, url, target_key, get_id=True):
         params = {
             'query': '{}'.format(filter_str)
         }
-        resp = self._get_request(url, params, self._headers, False)
-
-        if not self._check_http_response(resp):
-            return False
-
-        response = resp.json()
-        if not response:
+        try:
+            resp = self._get_request_retry(url, params, self._headers, False)
+            response = resp.json()
```

Comment:
can we do this outside of the try/except block? and just have the block be:
```
try:
    resp = self._get_request_retry(url, params, self._headers, False)
except OutputRequestFailure:
    return False
```
I think there a few different places that could be changed to move this outside of the block